update.go

```diff
@@ -147,7 +147,7 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				}
 
 			// If stopped, start Tailscale.
-			case ipn.Stopped.String():
+			case ipn.Stopped:
 				return m, func() tea.Msg {
 					err := libts.Up(ctx)
 					if err != nil {
```

Comment:
other nice thing you can do here is new lexical scope for such cases. nice tidbit.
```suggestion
					if err := libts.Up(ctx); err != nil {
```

Reply:
in this case i prefer the separate declaration because it matches how error handling is done elsewhere and makes it easier to spot that `libts.Up` is a mejor operation